src/Select.js

```diff
@@ -335,35 +359,18 @@ export default class Select extends Component<Props, State> {
 
     // scroll the focused option into view if necessary
     if (
-      this.menuRef &&
+      this.menuListRef &&
```

Comment:
At one point this referenced the menu. It's now a child container that actually scrolls called `MenuList` -- just updated for cleanliness.